server/lib/ses_Client.js

```diff
@@ -0,0 +1,6 @@
+import { SESClient } from '@aws-sdk/client-ses';
+const config = {
+  region: 'us-west-1',
+};
+const sesClient = new SESClient(config);
+export { sesClient };
```

Comment:
It's nicer to combine these and just write:
`export const sesClient = new SESClient(config);`

The code in here is so short now, that it doesn't seem like you even need to have this in a separate file. You could just put it at the top of server.ts.